config/processCcd.py

```diff
@@ -37,3 +38,7 @@
     config.calibrate.astrometry.solver.useWcsRaDecCenter = False  # It's off for some reason dunno yet
     config.calibrate.astrometry.solver.useWcsParity = False  # I doubt I guess right
     config.calibrate.astrometry.solver.useWcsPixelScale = False  # DGM says it's 0.4, but....
+else:
```

Comment:
Were there actually Sloan filters available?  Same as above, it seems like master should have a clean config and we can add on in a branch as new features are available.